### DISCUSSION TITLE
Default options for Cookie plugin

### DISCUSSION THREAD

**Discussion by Ridzhi** [quasarframework/quasar#9822] [General - Components / Directives / etc]
Is it possible?

**Comment by ontwikkelfabriek** [comment]
@Ridzhi 

You could define a (global) config options object that you will always pass to the `$q.cookie.set`:


```javascript
import cookieOptions from 'src/config/cookies'
...
$q.cookies.set('cookie_name', cookie_value, cookieOptions ) 
```

A bit more refined would be to wrap the `cookies.set` in your own `setCookie` function that will always use the default `cookieOptions`


```javascript
// e.g. src/utils.js 
import { Cookies } from 'quasar'
import cookieOptions from 'src/config/cookies'

const setCookie= (key,value) => {
 Cookies.set('quasar', 'framework', cookieOptions  )
}

export { setCookie }
```

Or you could go all the way and change the Cookie prototype ( with defineProperty) so it will always be created with your default options.

**Reply by Ridzhi** [reply]
thanks, I just expect something like `quasar.conf.js.framework.config.Cookie:{}`
